cmd/controller-manager/main.go

```diff
@@ -60,6 +61,11 @@ func main() {
 	c := manager.NewFederatedTypeConfigController(config, si)
 	c.Run(stopChan)
 
+	err = rspcontroller.StartReplicaSchedulingPreferenceController(config, config, config, stopChan, true)
```

Comment:
(No action required) I would like to see execution of this controller configurable before alpha, as per #101.

Reply:
Agree, I will take this up as soon as I finish #92, which is the next priority item in my list.